stan-dev/stan
# Issue/2706 cleanup docs

#### Submission Checklist


#### Summary
Removes directory `src/docs` and all make tasks for documentation.

#### Intended Effect
Keep all user-facing documentation in repo `docs`.  This simplifies build and testing requirements for this repo.

#### How to Verify
Check makefile tasks.  Check that there are no longer make tasks `make manual`, `make docs`, `make clean-manual`, `make clean-docs`

#### Copyright and Licensing

Please list the copyright holder for the work you are submitting (this will be you or your assignee, such as a university or company):

Columbia University

By submitting this pull request, the copyright holder is agreeing to license the submitted work under the following licenses:
- Code: BSD 3-clause (https://opensource.org/licenses/BSD-3-Clause)
- Documentation: CC-BY 4.0 (https://creativecommons.org/licenses/by/4.0/)
